js/pics/index.html

```diff
@@ -0,0 +1,18 @@
+
+<h2>Index</h2>
+
+<form data-ng-submit="indexVm.create()">
+  <input placeholder="Author" data-ng-model="indexVm.newPic.author" />
+  <input placeholder="Body" data-ng-model="indexVm.newPic.body" />
+  <input placeholder="Photo Url" data-ng-model="indexVm.newPic.photo_url" />
+  <button type="submit">Create</button>
+</form>
+
+<div data-ng-repeat="pic in indexVm.pics">
+  <a data-ui-sref="PicShow({id: $index})">
```

Comment:
Make sure the state name here matches exactly what you have in `app.js`. Looks like this one is capitalized, but the corresponding one in `app.js` is not.